src/Symfony/Component/Finder/Finder.php

```diff
@@ -440,6 +440,22 @@ public function sortByName(bool $useNaturalSort = false): static
         return $this;
     }
 
+    /**
+     * Sorts files and directories by name case insensitive.
+     *
+     * This can be slow as all the matching files and directories must be retrieved for comparison.
+     *
+     * @return $this
+     *
+     * @see SortableIterator
+     */
+    public function sortByCaseName(bool $useNaturalSort = false): static
```

Comment:
This feature looks nice (thanks @hmoreau) but I'm not sure about this method name. To me, "sort by case name", reads like you are referring to the name of the casing ("lowercase", "uppercase", "titlecase", etc.)

A less confusing name could be `sortByNameInsensitive()` or `sortByNameNatural()` (`natsortByName()`?)

Reply:
`sortByCaseInsensitiveName` would actually keep words in the order expected by English